lib/checkleakautovar.cpp

```diff
@@ -75,6 +77,16 @@ static bool isAutoDealloc(const Variable *var)
     return true;
 }
 
+static bool isVarTokComparison(const Token * tok, const Token ** vartok,
+                               const std::vector<std::pair<std::string, std::string>>& ops)
+{
+    for (auto op : ops) {
```

Comment:
In general I think it would be better to use the type `const auto &` when it will be a nontrivial type.. do you think it's better to take it by value than by reference here? The `const` shouldn't have a cost at least.

Reply:
No, I agree with `const auto &`.